framework/source/class/qx/ui/core/Widget.js

```diff
@@ -3687,6 +3687,19 @@ qx.Class.define("qx.ui.core.Widget",
     },
 
 
+    /**
+     * Return the ID of the current widget if it is a created as a 
```

Comment:
OK, then this:

> If the widget has been created as a ChildControl, return the id (name) of the ChildControl.

the problem is that your current sentence does not really work ... it is missing words.